pandas/tseries/frequencies.py

```diff
@@ -542,14 +542,16 @@ def get_standard_freq(freq):
     """
     Return the standardized frequency string
     """
-    if freq is None:
-        return None
+
+    msg = ("get_standard_freq(freq) is deprecated. "
+           "Use freq.rule_code instead if freq is a DateOffset object, "
```

Comment:
just recommend ``.to_offset``. (which accepts all of these). BTW, can we enhance the doc-string of ``.to_offset``? (add Parameters / more examples)